functions/controllers/user/getAuthUserCredentials.js

```diff
@@ -0,0 +1,39 @@
+const HttpStatus = require('http-status-codes');
+const { message, status } = require('../../util/constants');
+
+const { BAD_REQUEST, OK, INTERNAL_SERVER_ERROR } = HttpStatus;
+const { error, success } = status;
+const { somethingWentWrong } = message;
+
+const mapUsers = async (res, users) => {
+    const user = users.data();
+    return res.status(OK).json({ data: user, status: success });
+};
+
+const getUser = async (req, res, db) => {
+    const userName = req.user.userName.toLowerCase();
+    const users = await db
+        .collection('users')
+        .doc(userName)
+        .get();
+    if (!users) {
+        return res
+            .status(BAD_REQUEST)
+            .json({ message: somethingWentWrong, status: error });
+    }
+    return mapUsers(res, users);
+};
+
+const getAuthUserCredentials = async (req, res, db) => {
+    try {
+        return getUser(req, res, db);
+    } catch (errors) {
+        return res
+            .status(INTERNAL_SERVER_ERROR)
+            .json({ message: somethingWentWrong, status: errors });
+    }
+};
```

Comment:
This function is just returning the `getUser` function above.  i don't its doing anything.

Reply:
eslint rules stopping my functions from exceeding 20 lines so it call =s getuser and catches error so underneath 20 lines